### DISCUSSION TITLE
Can't seem to build and start caMicroscope distribution docker instance to run the platform on my Ubuntu 22.04 Machine

### DISCUSSION THREAD

**Discussion by Ronnie-Leon76** [camicroscope/caMicroscope#668] [Q&A]
I'm getting the error shown in the snapshot below. It's associated with the dicomsrv service defined in the develop.yml. I looked at the issues of the distro repo and @birm advised to use the following docker command: ```docker-compose -f develop.yml up``` instead of using the caMicroscope.yml file but I'm getting an issue. The error is associated with the orthanc_indexer microservice that was added. I'll appreciate any help on how to get around this. I would like to make improvements to caMicroscope by adding support for multi-channel image support
![Screenshot from 2024-01-20 12-04-02](https://github.com/camicroscope/caMicroscope/assets/60542881/2f32361c-60a1-4e87-a2cc-e381352504c1)
![Screenshot from 2024-01-20 12-03-12](https://github.com/camicroscope/caMicroscope/assets/60542881/1eac1736-342e-4be6-9de2-2c76f295dadb)

**Comment by Ronnie-Leon76** [comment]
Funny enough I have set up the same project on Gitpod and it works fine but it just doesn't work on my machine. The bug must be me, I got to find it. But it seems it is an issue with the Boost C++ Library

**Comment by birm** [comment]
I'll see if I can replicate the issue. Are you using develop.yml? What cpu architecture?

**Comment by Ronnie-Leon76** [comment]
I'm using develop.yml file. I'm running the docker instance on a 64-bit
architecture with 4 processors. I managed to setup the container on Gitpod
and I did interact with the web platform. I added several Whole Slide
Images and tried annotation and other features.

**Reply by birm** [reply]
I see the end of a cmake fail message in your first screenshot, can you add the whole log?

**Comment by birm** [comment]
@Ronnie-Leon76 We've tried some fixes; does your issue persist?

**Reply by Ronnie-Leon76** [reply]
![Screenshot from 2024-02-12 05-13-17](https://github.com/camicroscope/caMicroscope/assets/60542881/cf65e825-7a71-4302-9109-1d3e4c160e51)
![Screenshot from 2024-02-12 05-12-44](https://github.com/camicroscope/caMicroscope/assets/60542881/6fb0bd74-3a99-419d-93d8-024682c6261c)
![Screenshot from 2024-02-12 05-12-35](https://github.com/camicroscope/caMicroscope/assets/60542881/9130629c-b1fa-45d9-a97c-49a36ce983bd)

Still the same issue. I have an image with the full CMake error log. There is an issue with the library used in the Orthanc_indexer microservice

**Reply by Ronnie-Leon76** [reply]
Sorry for the late reply. I have been busy with some work.

**Reply by Ronnie-Leon76** [reply]
It is so funny that it works on Gitpod but not on my machine.

![image](https://github.com/camicroscope/caMicroscope/assets/60542881/57648b58-1416-440a-a41b-5d55be875679)

![Screenshot from 2024-02-12 21-18-17](https://github.com/camicroscope/caMicroscope/assets/60542881/8d627aa1-ec2e-4b4d-b6d5-fa7d7c54a670)

I have been using Gitpod to go through the codebase to understand the various components. I have been going through resources on multi-channel support since my interest is in image processing but I see there is an image classification feature that I can implement to improve caMicroscope digital pathology slide viewing web platform.

**Comment by YashKumarVerma** [comment]
give me few hours, will tirage on my local machine
